### DISCUSSION TITLE
历史话题支持批量清理

### DISCUSSION THREAD

**Discussion by SystemDev01** [CherryHQ/cherry-studio#1848] [Ideas]
![image](https://github.com/user-attachments/assets/f601da7d-87e0-4513-af4f-4d182ae7ec4e)
历史话题能否支持批量清理，并且这里面有大量的默认话题，点进去都是空的，这个没必要留着

**Comment by kangfenmao** [comment]
有没有什么操作交互上的建议

**Comment by SystemDev01** [comment]
![image](https://github.com/user-attachments/assets/4e9ae72f-7a42-4cdc-a62a-21c9cd024f43)
分组建议可以参考windows的目录分组，然后每个分组都有一个清理按钮可以清除整个分组的对话，可以加一个确定按钮以防误删，像一些空白对话，啥内容都没有的就直接过滤或者丢弃吧，不应该显示出来

**Comment by sjhsbhqf** [comment]
> 有没有什么操作交互上的建议

@kangfenmao 
我建议双击这个扫帚就能直接清空当前话题，
![image](https://github.com/user-attachments/assets/2ead80cf-3dea-4502-bb74-bad4de2518f0)

而不是说需要鼠标在不同位置点击两下
![image](https://github.com/user-attachments/assets/d4b4417b-ceb6-493e-b8a4-5cd9d95de21e)

当然原来的做法，也可以延续。不过还是添加一个更加快捷的方式吧，我可不喜欢存储聊天记录，这些是不是会占用webdav流量？
